src/renderer/components/+network-services/service-details.tsx

```diff
@@ -77,6 +77,24 @@ export class ServiceDetails extends React.Component<Props> {
           {spec.clusterIP}
         </DrawerItem>
 
+        {service.getClusterIps().length > 0 && (
+          <DrawerItem name="Cluster IPs">
+            {service.getClusterIps().join(", ")}
+          </DrawerItem>
+        )}
```

Comment:
Nit, `DrawerItem` has a `hidden` prop, which I think should be used instead of doing the `&&`.

Furthermore, I think it would be better to encapsulate all the IPs in `<Badge>` instead of just joining with `", "`. This would also apply to the above item too.

Reply:
How about just putting those in separate lines?

Reply:
That could work too, I just felt that badge's would look nicer.

Reply:
thank you, fixed it, but still have problems with test timeout.
```Timeout - Async callback was not invoked within the 60000 ms timeout specified by jest.setTimeout.Timeout - Async callback was not invoked within the 60000 ms timeout specified by jest.setTimeout.Error:```

Reply:
@vshakirova If the tests pass locally on your machine, I would just rerun them. If you don't have permissions, just poke one of us.